mcs/class/System.Security/Test/System.Security.Cryptography.Pkcs/AlgorithmIdentifierTest.cs

```diff
@@ -50,7 +50,6 @@ public void ConstructorEmpty ()
 			Assert.AreEqual (0, ai.KeyLength, "KeyLength");
 			Assert.AreEqual (defaultName, ai.Oid.FriendlyName, "Oid.FriendlyName");
 			Assert.AreEqual (defaultOid, ai.Oid.Value, "Oid.Value");
-			Assert.AreEqual (0, ai.Parameters.Length, "Parameters");
```

Comment:
why did this and the other tests need a change? if it's expected we should update the tests to assert for the new length instead.

Reply:
There is no `Parameters ` property in CoreFX implementation.

Reply:
Ok, so that's another API break since it's there in .NET Framework: https://msdn.microsoft.com/en-us/library/system.security.cryptography.pkcs.algorithmidentifier.parameters%28v=vs.110%29.aspx?f=255&MSPPError=-2147217396

Could you please file an issue on corefx to find out why?

Reply:
Yes, we should understand any non-obvious API changes :)

Reply:
Filed https://github.com/dotnet/corefx/issues/29706 for `Parameters` property on AlgorithmIdentifier type.

Reply:
Filed https://github.com/dotnet/corefx/issues/29708 for API break on CmsRecipientCollection

Reply:
@akoeplinger Could you please take a look at the response from corefx folks? ^^^

Reply:
@marek-safar @akoeplinger Could you please cherry-pick the changes from https://github.com/dotnet/corefx/pull/29880 ?

Reply:
There is still no any change for `Parameters` property on AlgorithmIdentifier type. I asked corefx folks about which decision they made.

Reply:
@MaximLipnin done